packages/atlas/src/components/_video/VideoPlayer/VideoOverlay.tsx

```diff
@@ -31,28 +34,41 @@ export const VideoOverlay: FC<VideoOverlayProps> = ({
   isPlayNextDisabled,
   isMinimized,
   playRandomVideoOnEnded = true,
+  currentVideoTimestamp,
 }) => {
   const [randomNextVideo, setRandomNextVideo] = useState<BasicVideoFieldsFragment | null>(null)
-  const { videos } = useBasicVideos(
+  const { videos, refetch } = useBasicVideos(
     {
+      limit: 1,
+      orderBy: VideoOrderByInput.ChannelCreatedAtAsc,
       where: {
+        ...publicVideoFilter,
         channel: {
           id_eq: channelId,
         },
+        createdAt_gt: currentVideoTimestamp,
```

Comment:
The name `currentVideoTimestamp` is really misleading. At first glance, I thought you mean the current time of the actual video(in the video player). Can we rename this to something more intuitive - maybe even `currentVideoCreatedAt`?

Reply:
agree, should be easier to match to `video.createdAt` prop this way